dojo/tools/dependency_track/parser.py

```diff
@@ -242,16 +345,15 @@ def get_description_for_scan_types(self, scan_type):
         return "The Finding Packaging Format (FPF) from OWASP Dependency Track can be imported in JSON format. See here for more info on this JSON format."
 
     def get_findings(self, file, test):
-
         # Exit if file is not provided
         if file is None:
             return list()
 
         # Load the contents of the JSON file into a dictionary
         data = file.read()
         try:
-            findings_export_dict = json.loads(str(data, 'utf-8'))
-        except:
+            findings_export_dict = json.loads(str(data, "utf-8"))
+        except BaseException:
```

Comment:
```suggestion
        except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8282/commits/2de95f283b3df8d72cc1577b35f2e202386dd84c